File: imparPar/imparPar java/imparPar.java
import java.util.Scanner;

public class ImparPar {
    public static void main(String[] args) {
        int num, resultado;

        Scanner dado = new Scanner(System.in);

        System.out.println("Digite um número:");
        num = dado.nextInt();

        resultado = num % 2;

        if (resultado == 0) {
            System.out.println("Esse número é par");
        } else {
            System.out.println("Esse número é ímpar");
        }
    }
}

File: imparPar/ImparPar C++/imparPar.cpp
/******************************************************************************

Welcome to GDB Online.
GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
Code, Compile, Run and Debug online from anywhere in world.

*******************************************************************************/
#include <stdio.h>

int main() {
    int num, resultado;

    printf("Digite um número: ");
    scanf("%d", &num);

    resultado = num % 2;

    if (resultado == 0) {
        printf("%d é um número par\n", num);
    } else {
        printf("%d é um número ímpar\n", num);
    }

    return 0;
}